klayout/drc/rule_decks/main.drc

```diff
@@ -14,880 +14,766 @@
+  report('DRC Run Report at', report_path)
 end
 
 #================================================
 #------------------ SWITCHES --------------------
 #================================================
-logger.info("Evaluate switches.")
+logger.info('Evaluate switches.')
 
 # connectivity rules
-if $conn_drc == "true"
+if $conn_drc == 'true'
   CONNECTIVITY_RULES = $conn_drc
-  logger.info("connectivity rules are enabled.")
+  logger.info('connectivity rules are enabled.')
 else
   CONNECTIVITY_RULES = false
-  logger.info("connectivity rules are disabled.")
-end # connectivity rules
+  logger.info('connectivity rules are disabled.')
+end
 
 # WEDGE
-if $wedge == "false"
-  WEDGE = $wedge
-else
-  WEDGE = "true"
-end # WEDGE
+WEDGE = if $wedge == 'false'
+          $wedge
+        else
+          'true'
+        end
 
-logger.info("Wedge enabled  %s" % [WEDGE])
+logger.info(format('Wedge enabled  %s', WEDGE))
 
 # BALL
-if $ball == "false"
-  BALL = $ball
-else
-  BALL = "true"
-end # BALL
+BALL = if $ball == 'false'
+         $ball
+       else
+         'true'
+       end
 
-logger.info("Ball enabled  %s" % [BALL])
+logger.info(format('Ball enabled  %s', BALL))
 
 # GOLD
-if $gold == "false"
-  GOLD = $gold
-else
-  GOLD = "true"
-end # GOLD
+GOLD = if $gold == 'false'
+         $gold
+       else
+         'true'
+       end
 
-logger.info("Gold enabled  %s" % [GOLD])
+logger.info(format('Gold enabled  %s', GOLD))
 
-if $mim_option
-  MIM_OPTION = $mim_option
-else
-  MIM_OPTION = "B"
-end
+MIM_OPTION = $mim_option || 'B'
 
-logger.info("MIM Option selected %s" % [MIM_OPTION])
+logger.info(format('MIM Option selected %s', MIM_OPTION))
 
 # OFFGRID
-if $offgrid == "false"
-  OFFGRID = false
-else
-  OFFGRID = true
-end # OFFGRID
+OFFGRID = $offgrid != 'false'
 
-logger.info("Offgrid enabled  %s" % [OFFGRID])
+logger.info(format('Offgrid enabled  %s', OFFGRID))
 
 if $thr
-    threads($thr)
+  threads($thr)
 else
-    thr ||=  Etc.nprocessors
-    threads(thr)
+  thr ||= Etc.nprocessors
+  threads(thr)
 end
-logger.info("Number of threads to use %s" % [$thr])
+logger.info(format('Number of threads to use %s', $thr))
 
 #=== PRINT DETAILS ===
 logger.info("Verbose mode: #{$verbose}")
-if $verbose == "true"
+if $verbose == 'true'
   verbose(true)
 else
   verbose(false)
 end
 
 # === TILING MODE ===
-if $run_mode == "tiling"
+case $run_mode
+when 'tiling'
   tiles(500.um)
   tile_borders(10.um)
-  logger.info("Tiling  mode is enabled.")
+  logger.info('Tiling  mode is enabled.')
 
-elsif $run_mode == "deep"
+when 'deep'
   #=== HIER MODE ===
   deep
-  logger.info("deep  mode is enabled.")
+  logger.info('deep  mode is enabled.')
 else
   #=== FLAT MODE ===
   flat
-  logger.info("flat  mode is enabled.")
-end # run_mode
+  logger.info('flat  mode is enabled.')
+end
 
 # METAL_TOP
-if $metal_top
-  METAL_TOP = $metal_top
-else
-  METAL_TOP = "9K"
-end # METAL_TOP
+METAL_TOP = $metal_top || '9K'
```

Comment:
any reason we don't keep using `$metal_top` here?

Reply:
`METAL_TOP` variable is a different than `metal_top`. It's used in other tables.

Reply:
Referring to #37

Reply:
Agreed let's keep any config refactoring for #37?